core/src/integration-test/java/com/scalar/db/storage/cassandra/CassandraWithReservedKeywordsIntegrationTest.java

```diff
@@ -0,0 +1,25 @@
+package com.scalar.db.storage.cassandra;
+
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.storage.StorageIntegrationTestBase;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+@SuppressFBWarnings("ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD")
+public class CassandraWithReservedKeywordsIntegrationTest extends StorageIntegrationTestBase {
```

Comment:
I think we should add new integration tests for reserved keywords rather than extending `StorageIntegrationTestBase` because we don't need to run all the tests in `StorageIntegrationTestBase` to check if reserved keywords work or not. We should have minimum integration tests to check it.

Also, I think we should run the integration tests for reserved keywords not only for the Cassandra adaptors but also all the adaptors. So can you please add tests for all the adaptors? Thanks.

Reply:
I fixed the reserved keyword issue and added tests for other storage adapters as well. Please take a look again when you have time.